### ISSUE TITLE
All map style/icons/priorities/colors issues are collected here

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#1109]
Let's collect here all separate issues (and write new ones).
(please check [the docs](https://github.com/organicmaps/organicmaps/blob/master/docs/STYLES.md) before starting work on these issues)

- [ ] #11660
- [x] #988 
- [x] #949 
- [x] #873
- [x] #1253
- [x] #684
- [x] #1919
- [x] #1995
- [x] #1414
- [x] #755
- [x] #2162
- [x] #2170
- [x] #2186

**Comment by VladimirMorozov** [comment]
Should I create a separate issue and then link it here or just write it here?

* nearby POI icons hide each other. It's impossible to discover hidden POIs without zooming in everywhere. See screenshots.

![C44DF231-A031-4DD5-841F-E3E9075C02AE](https://user-images.githubusercontent.com/2279813/132118219-e70d8b85-d06d-4a42-b43f-ef1678600511.jpeg)

Zoom in and see a hidden shop

![5971BC33-CC53-40DB-9C7E-F6B70511AB02](https://user-images.githubusercontent.com/2279813/132118230-d447be37-4a84-4d39-81ec-dff2e232891c.jpeg)


Possible solutions:
* Group conflicting icons (display them in a row or grid)
* Conflicting icons become small colored dots - then the user at least knows that he should zoom in to see more
* icons overlap, but consistently - no matter how zoom one icon is always on top of the other, you see that there is something behind it.


Btw, maybe this should be pinned like UX grouping issue.

**Comment by biodranik** [comment]
Thanks, this is an old problem, we can track it here. Regarding a proper solution, overlapping icons should not only be somehow visible but also allow to select them easily. I'm in favor of something simple and easy for users. Maybe simply decreasing icons padding would already help tremendously. 

Do you know good examples from other maps? Google Maps, for example, does the same as what we do in OM.

**Comment by VladimirMorozov** [comment]
Overlapping icons could be more easily selected by zooming in. Important is that the user will know that but zooming in there is more to see and maybe click on. At least for me this solution will suffice (or any other I 've mentioned).

Place on screenshots is at 35.380906, 24.581798

**Comment by strump** [comment]
Here's how Apple Maps (iPad) solved overlapping icons issue:
https://user-images.githubusercontent.com/720808/134462447-f7c6a611-36de-4560-a872-9681313a0c30.mp4

If there is no place for icon then only small dot is display with no label. But this dot is still clickable. User can see there's some POI and can zoom in to get full icon with label.

**Comment by biodranik** [comment]
On the video, some icons are still hidden. Showing dots may work if ALL dots will be visible.

**Comment by arnaudvergnet** [comment]
Vespucci shows all icons overlapping, and displays a popup when clicking an area with a lot of poi.

![Screenshot_20210924-180612_Vespucci](https://user-images.githubusercontent.com/80701113/134706672-be7ff4b3-9d90-4335-a6b9-4a18e37b25f5.png)

Streetcomplete shows small dots but those are not selectable : you either zoom more or complete the quests around.

![Screenshot_20210924-180836_Street�Complete](https://user-images.githubusercontent.com/80701113/134707073-471c276f-faf7-4c72-a2ac-6d426a1f184f.png)

OsmAnd is a mix of Vespucci and OM, which IMO is the most straightforward for users. Not all poi are shown when zoomed out, and they appear when close. If there is still no room, the names are hidden and only icons show. If 2 poi are still too close to make selection easy, a popup is shown like in Vespucci to let the user select.

| Zoomed out | Zoomed in |
| - | - |
| ![Screenshot_20210924-181316_OsmAnd~](https://user-images.githubusercontent.com/80701113/134707917-83749121-4f11-4cc5-8d04-97834725fc33.png) | ![Screenshot_20210924-181259_OsmAnd~](https://user-images.githubusercontent.com/80701113/134707921-91d66d3e-ea19-43e1-bfa7-09897b6504b2.png) |

**Comment by patepelo** [comment]
I would close this and open a dedicated issue for subway entrances if the problem still exists.
